tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = ui-tests, flake8
+skipsdist = True
+
+[testenv:ui-tests]
+passenv = DISPLAY
+deps = -rfrontend/test/ui/requirements/requirements.txt
+commands =
+    sh bin/circleci/run-integration-tests.sh
+    pytest -v --driver=Firefox --html=results.html frontend/test/ui {posargs}
```

Comment:
You should use `--self-contained-html` so you don't need to copy all the assets.